FranDepascuali/Wolox-Training-App
# Log in view models controllers

## Sumary

**Trello cards**:
tech-login: https://trello.com/c/CyFcx2ll/21-tech-login
login: https://trello.com/c/Uv3haBcZ/14-login

Developed log-in view controllers and models.
## Test Plan
- Open the app
- Login with username 'f@wolox.com.ar' password 'probando'
- You should see a spinner while logging in.
- The spinner should disappear when logged.
- If mail or password aren't correct, a toast with label "error" will appear. 
- If signup button is clicked, it leads to signUp form.
## Screenshots

![login](https://cloud.githubusercontent.com/assets/12101394/7593989/2943c80a-f8b4-11e4-8aa2-8e399afe206a.gif)
## Known issues
- Errors are not yet classified.
- News controllers and model not yet implemented, successful log in will lead to news controller.
